src/app/core/session/session-service/synced-session.service.ts

```
@@ -144,12 +153,16 @@ export class SyncedSessionService extends SessionService {
   private updateLocalUser(password: string) {
     // Update local user object
     const remoteUser = this._remoteSession.getCurrentUser();
-    this._localSession.saveUser(remoteUser, password);
+    if (remoteUser) {
+      this._localSession.saveUser(remoteUser, password);
+    }
   }
 
-  private startSync() {
+  private startSync(): Promise<any> {
     // Call live syn even when initial sync fails
-    return this.sync().finally(() => this.liveSyncDeferred());
+    return this.sync()
+      .catch(() => {})
```

is this good to silently ignore any errors here?